macOS/DuckDuckGo/AIChat/UserScript/AIChatUserScriptHandling.swift

```diff
@@ -19,30 +19,39 @@
 import UserScript
 
 protocol AIChatUserScriptHandling {
-    func handleGetUserValues(params: Any, message: UserScriptMessage) -> Encodable?
-    func openSettings(params: Any, message: UserScriptMessage) async -> Encodable?
+    func openAIChatSettings(params: Any, message: UserScriptMessage) async -> Encodable?
+    func getAIChatNativeConfigValues(params: Any, message: UserScriptMessage) -> Encodable?
+    func closeAIChat(params: Any, message: UserScriptMessage) -> Encodable?
```

Comment:
`Subfeature.Handler` is an `async throws` closure, so you can define all your methods in the protocol as such (I do it in my user scripts anyway), and then you’d avoid using `Task` in `closeAIChat`.

Reply:
Good catch, thanks!